certificate/database.go

```diff
@@ -459,6 +473,46 @@ func getCurrentTrust(certID, issuerID int64) (int64, error) {
 	return trustID, nil
 }
 
+func getCurrentTrustIDForCert(certID int64) (int64, error) {
+
+	var trustID int64
+
+	row := db.QueryRow("SELECT id FROM trust WHERE cert_id=$1 AND is_current=TRUE", certID)
+
+	err := row.Scan(&trustID)
+
+	if err != nil {
+
+		if err == sql.ErrNoRows {
+			return -1, nil
```

Comment:
I wanted to handle the NoRows error inside the database functions and -1 without an error seemed like good way to say not found without having the caller handling an error.
It might be better to have some constants though.